grpc-spring-boot-starter/src/main/java/org/lognet/springboot/grpc/tracing/TracingClientInterceptor.java

```diff
@@ -0,0 +1,33 @@
+package org.lognet.springboot.grpc.tracing;
+
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.ForwardingClientCall;
+import io.grpc.Metadata;
+import io.grpc.MethodDescriptor;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.TraceId;
+
+public class TracingClientInterceptor implements ClientInterceptor {
```

Comment:
Should it be moved to grpc-client-spring-boot-starter?

Reply:
Yes

Reply:
done